README.md

```diff
@@ -24,6 +24,18 @@ Clone the repository:
 $ git clone https://github.com/guidance-guarantee-programme/pension_guidance.git
 ```
 
+Install PrinceXML:
+
+```sh
+$ http://www.princexml.com/download/
+```
+
+Install and start Redis:
+
+```sh
+$ brew install redis && redis-server /usr/local/etc/redis.conf
```

Comment:
fair point - for some reason I couldn't get the background service to run, but i'll alter it to that